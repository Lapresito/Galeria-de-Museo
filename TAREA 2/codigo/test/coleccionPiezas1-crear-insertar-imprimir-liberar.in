crearColeccionPiezas
imprimirColeccionPiezas
liberarColeccionPiezas
crearColeccionPiezas
crearFecha 1/1/2001
crearPieza 7 LaxaAristaTriste Carlos Nelson
agregarColeccionPiezas
imprimirColeccionPiezas
liberarColeccionPiezas
crearColeccionPiezas
crearFecha 1/1/2001
crearPieza 7 SacraBarcaTriste Carlos Nelson
agregarColeccionPiezas
crearFecha 1/1/2001
crearPieza 10 RaraEspadaDesquiciada Luciana Diaz
agregarColeccionPiezas
crearFecha 1/1/2001
crearPieza 6 MalditaMangostaFunebre Luciana Diaz
agregarColeccionPiezas
crearFecha 1/1/2001
crearPieza 2 AltaAristaTriste Luciana Diaz
agregarColeccionPiezas
crearFecha 1/1/2001
crearPieza 16 RaraRecursionFurtiva Luciana Diaz
agregarColeccionPiezas
imprimirColeccionPiezas
liberarColeccionPiezas
Fin
