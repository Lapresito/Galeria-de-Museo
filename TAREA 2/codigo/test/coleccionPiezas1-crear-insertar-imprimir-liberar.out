Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Coleccion de piezas creada de forma exitosa.
2> 3> Coleccion liberada con exito.
4> Coleccion de piezas creada de forma exitosa.
5> Fecha creada en forma exitosa.
6> Pieza creada en forma exitosa.
7> Pieza agregada de forma exitosa.
8> Pieza: 7
LaxaAristaTriste-Carlos Nelson
Creada en: 1/1/2001
9> Coleccion liberada con exito.
10> Coleccion de piezas creada de forma exitosa.
11> Fecha creada en forma exitosa.
12> Pieza creada en forma exitosa.
13> Pieza agregada de forma exitosa.
14> Fecha creada en forma exitosa.
15> Pieza creada en forma exitosa.
16> Pieza agregada de forma exitosa.
17> Fecha creada en forma exitosa.
18> Pieza creada en forma exitosa.
19> Pieza agregada de forma exitosa.
20> Fecha creada en forma exitosa.
21> Pieza creada en forma exitosa.
22> Pieza agregada de forma exitosa.
23> Fecha creada en forma exitosa.
24> Pieza creada en forma exitosa.
25> Pieza agregada de forma exitosa.
26> Pieza: 2
AltaAristaTriste-Luciana Diaz
Creada en: 1/1/2001
Pieza: 6
MalditaMangostaFunebre-Luciana Diaz
Creada en: 1/1/2001
Pieza: 7
SacraBarcaTriste-Carlos Nelson
Creada en: 1/1/2001
Pieza: 10
RaraEspadaDesquiciada-Luciana Diaz
Creada en: 1/1/2001
Pieza: 16
RaraRecursionFurtiva-Luciana Diaz
Creada en: 1/1/2001
27> Coleccion liberada con exito.
28> Fin.
